modules/os2forms_forloeb/src/Plugin/WebformHandler/MaestroNotificationHandler.php

```diff
@@ -240,6 +240,28 @@ private function getRecipientElements(): array {
       'cvr_value_element',
       'os2forms_person_lookup',
     ];
+
+    // Expand composite elements, NOT custom composite elements.
+    foreach ($elements as $key => $element) {
+      $orgElement = $this->getWebform()->getElement($key);
```

Comment:
What does “org” mean here? “Original”? “Organ”? “Organization”?

```suggestion
      $formElement = $this->getWebform()->getElement($key);
```

Reply:
Renamed.